src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import isEqual from 'lodash/isEqual';
+
+import {block} from '../utils/cn';
+import {FieldControl} from './components/FilterControl';
+import {FilterSelect} from './components/FilterSelect';
+import {ClearFiltersButton} from './components/ClearFiltersButton/ClearFiltersButton';
+
+import i18n from './i18n';
+
+import type {FilterSchema} from './schemas/types';
+
+import './Filters.scss';
+
+const b = block('filters');
+
+export type FilterValue<TFilterValue extends any = string[]> = TFilterValue;
+export type FiltersValue<TFiltersValue extends any = Record<string, FilterValue>> = TFiltersValue;
+
+/**
+ * Filters schema to retreive editors/initial values/other schema options.
+ */
+export type FiltersSchema = Record<string, FilterSchema<any, any>>;
```

Comment:
Why any?

Reply:
Basically, filters value could be any type. I didn't come up with better solutions, so I'm open to any suggestions :)

Reply:
`Unknown`?